### DISCUSSION TITLE
Mobile app is crashing for target version to 33

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#648] [General Discussion]
@swayangjit I have changed the apk target to 33 and published it. But now when i download the apk in android version below 12 it is working fine.
But when i download it in android version 13 the app is crashing and getting an error. I have attached the screenshot of the error. 
![Screenshot_2023-11-09-14-48-26-912_com miui bugreport](https://github.com/Sunbird-Ed/Community/assets/72926048/f9f871da-dcb1-4ee2-8b13-2f56b339245a)

**Comment by swayangjit** [comment]
@siddhishinde0723  Can you please try to remove "cordova-plugin-local-notification" plugin and add the following https://github.com/fquirin/cordova-plugin-local-notifications.git

**Reply by siddhishinde0723** [reply]
Okay ill change it and let you know.

**Reply by siddhishinde0723** [reply]
@swayangjit  Thanks the issue is be solved
